src/Orc.Controls/Controls/LogViewer/Models/ExpressionType.cs

```diff
@@ -0,0 +1,24 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ExpressionType.cs" company="WildGums">
+//   Copyright (c) 2008 - 2018 WildGums. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace Orc.Controls.Models
+{
+    public enum ExpressionType
```

Comment:
Same here, use a better name (e.g. LogFilterExpressionType) and Orc.Controls namespace.